src/plugins/home/public/application/components/homepage/hero_section/get_started.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { i18n } from '@osd/i18n';
+import { FormattedMessage } from '@osd/i18n/react';
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiTitle,
+  EuiImage,
+  EuiLink,
+  EuiIcon,
+} from '@elastic/eui';
+import { HeroSection } from './hero_section';
+import illustration from '../../../../assets/illustration.svg';
+import { getServices } from '../../../opensearch_dashboards_services';
+import logo from '../../../../assets/logos/chat.svg';
+
+export const GetStartedSection: React.FC<{ olly?: boolean }> = ({ olly = true }) => {
+  const services = getServices();
+  const getUrl = services.application.getUrlForApp;
+  const navigate = services.application.navigateToApp;
+  const prompts = services.homeConfig.prompts;
+  type Prompt = typeof prompts extends Array<infer T> ? T : never;
+
+  const OLLY_DESCRIPTION = (
```

Comment:
IMO this content is pretty nested and specific to o11y. if we wanted to have a new feature highlight then this would be a little bit rough.